kernel/runtime/crt-init/runtime/crtend.S

```diff
@@ -24,7 +24,7 @@
  * Copyright 2016 Randolf Rotta, Robert Kuban, and contributors, BTU Cottbus-Senftenberg
  */
 
-.section .init
```

Comment:
no, not here. this is userspace stuff where the init sections actually belongs to.